crates/typst-pdf/src/gradient.rs

```diff
@@ -63,12 +63,6 @@ pub(crate) fn write_gradients(ctx: &mut PdfContext) {
                     Quadrant::Fourth => (0.0, 1.0, cos, sin + 1.0),
                 };
 
-                let clamp = |i: f64| if i < 1e-4 { 0.0 } else { i.clamp(0.0, 1.0) };
-                let x1 = clamp(x1);
-                let y1 = clamp(y1);
-                let x2 = clamp(x2);
-                let y2 = clamp(y2);
-
```

Comment:
I could imagine that some pdf readers need the clamped values, but my testing capabilities are limited :/